tests/unit/components/Editor/ItemEditor.spec.js

```diff
@@ -14,6 +14,10 @@ describe("ItemEditor.vue", () => {
       type: "question",
       time: 20,
     },
+    {
+      type: "question",
+      time: 10,
```

Comment:
this timestamp should be greater than the timestamp of the item preceding it or else the order will be broken